typescript/src/Study-Lines-Add-Combos.ts

```diff
@@ -940,15 +940,15 @@ class LinePropertyAutoInput extends LinePropertyInput {
                 this.lineProperty.jsonId as number,
             ) >= 0
         ) {
-            visible.attr("eddautocompletetype", "User");
+            visible.attr("data-eddautocompletetype", "User");
```

Comment:
Similar to with `typescript/modules/EDDAutocomplete.ts`, this file is sort-of reverting, dropping the `data-` attribute prefix and using the JQuery `data` API instead of the `attr` API.